client/packages/common/src/api/GqlContext.tsx

```diff
@@ -43,6 +43,14 @@ interface ResponseError {
   extensions?: { details?: string };
 }
 
+export class StdError extends Error {
```

Comment:
Nit: rename to `GraphqlStdError` and maybe a quick comment what the fields are...